### PR TITLE
docs: otlp export

### PR DESCRIPTION
**Pull request by TBonnin**
## Describe your changes

Let me know what you think @bastienbeurier Do we want to go more in depth or is it enough doc for now?

Adding doc page for OTLP export feature

Screenshot 2024-11-12 at 13 57 38

## Issue ticket number and link
https://linear.app/nango/issue/NAN-2084/docs-for-otel-export

### CODE REVIEW

**Comment by bastienbeurier** [comment]
This is great! Thank you @TBonnin

**Review by bastienbeurier — APPROVED** [review]

File: docs-v2/features/otlp-export.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**Prerequisites**
+- An active Nango Scale plan subscription
+- Access to an OpenTelemetry collector
+</Info>
+
+### Supported Events
+The following integration events are currently exported (as of November 2024):
+- Sync executions
+- Action executions
+- Webhook executions
```
Comment (bastienbeurier) [inline docs-v2/features/otlp-export.mdx]: I would specify: 3rd-party API webhooks

File: docs-v2/features/otlp-export.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**Step 1**: Access your Environment settings
+
+Log into the Nango dashboard and navigate to the Environment settings page.
+
+**Step 2**: Configure OpenTelemetry Export
+
+In the Environment Settings, provide the following information:
+- OpenTelemetry endpoint: The endpoint URL of your OpenTelemetry collector. Ex: https://my.otlp.collector:4318/v1/
+- Headers (Optional): Any authentication headers or additional headers required to access your collector
```
Comment (bastienbeurier) [inline docs-v2/features/otlp-export.mdx]: I would specify that headers are usually required for authorization purposes with a Datadog example.

**Review by bodinsamuel — APPROVED** [review]

File: docs-v2/features/otlp-export.mdx (hunk 2 above)
Comment (TBonnin) [reply docs-v2/features/otlp-export.mdx]: Datadog agent doesn't have a default header. It is up to whoever is running the agent to secure its access

Comment (bastienbeurier) [reply docs-v2/features/otlp-export.mdx]: All good then!

**Pull request merged**